ptcg_cockatrice.xml

```diff
@@ -665,6 +1057,18 @@
 			<text>{3}{B}: Evolve into Weezing
 				Deathtouch</text>
 		</card>
+		<card>
+			<name>Weezing</name>
+			<set picURL="https://images.pokemontcg.io/base5/58_hires.png" rarity="common">PTCG</set>
+			<color>B</color>
+			<manacost>3BB</manacost>
+			<type>Pokémon - Poison</type>
+			<pt>3/3</pt>
+			<tablerow>2</tablerow>
+			<text>{X}{B}: Evolve from Koffing
```

Comment:
```suggestion
			<text>{X}{B}: Evolve from Koffing. Put X -1/-1 counters on up to X target Pokémon.
```

Reply:
gotcha thanks, I wasn't sure how to word it to still have X set but this makes sense